### PR TITLE
Project finished.

### PR DESCRIPTION
**Pull request by Daniduran-dev**
## Design Tear down

- Added the appropriate semantic HTML tags
- Added different semantic HTML layout
- Added different color shades of a particular color
- Added a combination of 2 different font types on a web page

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# HTML FORMS
+
+> Creating a Smashing Magazine Website
+
+![screenshot](images/screenshot.PNG)
```
Comment (majovanilla) [inline Readme.md:5]: - Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...

File: index.html [hunk 2]
```diff
@@ -93,40 +95,388 @@ <h2 class="box-header">Lorem Ipsum Generator</h2>
...
+          Massa ultricies mi quis hendrerit dolor. Morbi tincidunt augue interdum velit. 
+          Nibh mauris cursus mattis molestie a iaculis at erat pellentesque.
+        </p>
+
+        <p>
+          Plugins React Responsive Web Design Service Workers Sketch Typography UIUsabilityUser 
+          Experience Wall papersWeb Design WordPress Workflow
+        </p>
+      </div>
+    </footer>
```
Comment (majovanilla) [inline index.html]: - The footer should contain centered links. Just change the content to be links as the original page.

File: style.css [hunk 3]
```diff
@@ -57,102 +60,368 @@ html {
...
 }
+
 .box-3 {
   background-color: #64686b;
 }
+
 .box-4 {
   background-color: #5f6164;
 }
+
```
Comment (majovanilla) [inline style.css]: You have a repeated styling rules(box-3 and box-4)

**Review by majovanilla — CHANGES_REQUESTED** [review]
# Code Review in progress 👀 by Maria Reyes


Hey team, 👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...

- Please write meaningful pull request messages. Check this [example](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)

- Check other requirements below.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by uryelah** [comment]
Hello **Daniel Garcia Duran** & **Okiki Apelehin**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all, good job at fixing most of the required changes from the last code review 🎇**

There are still some issues to work through so hang in there 💪 

---------    

### Required changes 🛠 
---------

- [ ] The divs with the `.wrapper-content` class should be `<article>`s. Kindly change them to the semantic tag. You can read more about when to use the[ article tag here.](https://bitsofco.de/sectioning-content-in-html5/)

- [ ] The footer should contain centered links as in the[ original page](https://www.smashingmagazine.com/). Please check the image below for reference,
Screenshot_2020-03-16 Smashing Magazine — For Web Designers And Developers


- [ ] As the previous TSE mentioned you should add a professional and meaningful PR message as indicated by her link. Check the gif below for an example.
69548761-3a8d3600-0f98-11ea-9f19-14da361c12e2

- [ ] This is a design tear down and not a clone, please remove the icons or any color that you have in the site. You can represent them with grey divs. The idea is to represent the visual weight they have in the site visual hierarchy, color disrupts that as it calls too much attention to itself when around grey areas.

----

**Kindly fix the REQUIRED above issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by uryelah** [comment]
Hey guys, good work fixing the missing article tags and icons.

You are still missing two required changes above.
For your PR message, describe the things that you did building the clone in the bullet point format. Please read the missing changes carefully again bellow:

### Required changes 🛠 
---------

- [ ] The footer should contain centered links as in the[ original page](https://www.smashingmagazine.com/). Please check the image below for reference,
How the footer should look like:
Screenshot_2020-03-16 Smashing Magazine — For Web Designers And Developers

How your footer looks like:
Screenshot_2020-03-16 Smashing Magazine


- [ ] As the previous TSE mentioned you should add a professional and meaningful PR message as indicated by her link. Check the gif below for an example.
69548761-3a8d3600-0f98-11ea-9f19-14da361c12e2

----

**Kindly fix the all REQUIRED above issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by lekegitrepo** [comment]
### Hi @Daniduran-dev and @okikiola11, you have done a good job.
### This project is approved. Good luck with the next project.

**_Note: Please try fix to the merge conflict before submitting for review._**

### Review by: Temitayo Adeleke

-   Github: [@lekegitrepo](https://github.com/lekegitrepo)
-   Twitter: [@leketemi](https://twitter.com/leketemi)
-   Linkedin: [linkedin](https://www.linkedin.com/in/adeleke-temitayo-a69125188/)

**Review by lekegitrepo — APPROVED** [review]
### Status: Approved

**Pull request merged**